code/modules/mob/living/carbon/human/species/species_attack.dm

```diff
@@ -245,17 +245,29 @@
 	if(prob(25))
 		target.apply_effect(1, INCINERATE, 0)
 
-/datum/unarmed_attack/claws/vaurca_bulwark
-	attack_verb = list("punched", "clobbered", "lacerated")
-	attack_noun = list("clawed fists")
+/datum/unarmed_attack/vaurca_bulwark
+	attack_verb = list("punched", "pulverized", "hammered")
```

Comment:
"pulverized" is (afaik) generally used when something is destroyed, maybe use a different verb to avoid confusion?

Reply:
pulverised is used for several other unarmed attacks as is, so i don't see a need to remove it